datadog_checks_dev/datadog_checks/dev/tooling/commands/release/testable.py

```diff
@@ -67,15 +86,45 @@ def create_trello_card(client, teams, pr_title, pr_url, pr_body, dry_run):
 @click.command(
     context_settings=CONTEXT_SETTINGS, short_help='Create a Trello card for each change that needs to be tested'
 )
-@click.option('--start', 'start_id', help='The PR number or commit hash to start at')
-@click.option('--since', 'agent_version', callback=validate_version, help='The version of the Agent to compare')
+@click.argument('base_ref', callback=validate_version)
+@click.argument(
+    'target_ref', callback=functools.partial(validate_version, ignore=['origin/master']), default='origin/master'
```

Comment:
IMHO, we should force the user to provide both arguments. `target_ref` defaulting to master is only useful for the first release and might lead to wrong cards being created if used later on.

Reply:
Sounds legit, made `target_ref` required.